site/gatsby-site/i18n/locales/fr/incidents.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Incident List": "Liste des Incidents",
+    "This is a simple numeric listing of all incidents and their reports within the database. If you would like to explore the contents of the reports, you should work through the<1> Discover app</1>.": "Il s'agit d'une simple liste numérique de tous les incidents et de leurs rapports dans la base de données. Si vous souhaitez explorer le contenu des rapports, vous devez parcourir les<1> Découvrir les incidents</1>.",
```

Comment:
Please update to "Il s'agit d'une simple liste numérique de tous les incidents et de leurs rapports dans la base de données. Si vous souhaitez explorer le contenu des rapports, vous devez parcourir <1> Découvrir les incidents</1>."

Reply:
Done!